NEWS

```diff
@@ -178,12 +178,12 @@ CHANGES WITH 246 in spe:
           traffic). DataBitRate=, DataSamplePoint=, FDMode=, FDNonISO= have
           been added to configure various CAN-FD aspects.
 
-        * systemd-networkd's [DHCPv6] section gained a new WithoutRA= boolean
-          setting. If enabled, DHCPv6 will be attempted right-away without
-          requiring an Router Advertisement packet suggesting it
-          first. Conversely, the [IPv6AcceptRA] gained a boolean option
-          DHCPv6Client= that may be used to turn off the DHCPv6 client even if
-          the RA packets suggest it.
+        * systemd-networkd's [DHCPv6] section gained a new WithoutRA= setting.
+          If enabled, DHCPv6 will be attempted right-away without requiring an
+          Router Advertisement packet suggesting it first. Conversely, the
+          [IPv6AcceptRA] section gained a boolean option DHCPv6Client= that may
+          be used to turn off the DHCPv6 client even if the RA packets suggest
```

Comment:
I think "boolean" was making the whole clearer.

Reply:
Sorry, I cannot follow the above comments by @ssahani and @keszybz. Anyway, we usually merge commits about NEWS and provide better words in later commits. Please submit in later commits with better explanation. Sorry for my bad explanation.

Reply:
See a68da22257.

Reply:
Ugh, WithoutRA= is not a boolean anymore...